operator/src/test/java/org/keycloak/operator/OperatorE2ELocalRunTest.java

```diff
@@ -0,0 +1,51 @@
+package org.keycloak.operator;
+
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.dsl.Resource;
+import io.javaoperatorsdk.operator.config.runtime.DefaultConfigurationService;
+import io.javaoperatorsdk.operator.junit.OperatorExtension;
+import org.jboss.logging.Logger;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.keycloak.operator.v2alpha1.KeycloakController;
+import org.keycloak.operator.v2alpha1.crds.Keycloak;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+
+public class OperatorE2ELocalRunTest {
+    private static final Logger logger = Logger.getLogger(OperatorE2ELocalRunTest.class);
+
+    @RegisterExtension
+    OperatorExtension operator =
+            OperatorExtension.builder()
+                    .withConfigurationService(DefaultConfigurationService.instance())
+                    .waitForNamespaceDeletion(false)
```

Comment:
the problem with true is that somethimes namespace deletion takes forever and then the test doesnt pass

Reply:
> namespace deletion takes forever

If we are not using any specific finalizer this should not happen AFAIK, but I know it might be challenging to fix this in this PR ( [e.g.](https://github.com/lightbend/cloudflow/blob/7dad9e424c2ad89292fba5fbc24c6d6a2478e388/core/cloudflow-it/src/it/scala/ItSetup.scala#L97) ).

I'm happy to leave it as-is for now and open an Issue to follow up in the future.

Reply:
+1